lib/rules/empty-line-between-blocks.js

```diff
@@ -41,30 +103,90 @@ module.exports = {
   'detect': function (ast, parser) {
     var result = [];
 
-    ast.traverseByType(['ruleset'], function (ruleset, i, parent) {
+    // Set syntax
+    syntax = ast.syntax;
+
+    ast.traverseByType('ruleset', function (node, j, p) {
       var space;
 
-      space = findNearestReturn(parent, i);
+      if (syntax === 'scss') {
+        space = findNearestReturn(p, j);
```

Comment:
When the returned object contains a property named space I think this should probably be called something else.. `space.space` is a little confusing.